### PR TITLE
Bug/892

### LINKED ISSUE #892: Tag an image with e.g `rack/web` format.
Currently new docker images are tagged with a hash. But that could run into issues if different apps have same process names and build context.

Would be nice to revert the naming format to use e.g `rack/web`.

### PR DESCRIPTION
**Pull request by awsmsrc**
fixes #892 

@MiguelMoll can you test this one?

### CODE REVIEW

**Comment by MiguelMoll** [comment]
Code looks good and it solved the problem I was experiencing. :+1: 

rack images tagged as expected:

```
rack/registry
node-workers/web
node-workers/worker
rack/monitor
rack/web
httpd/web
httpd
golang
node
```

**Comment by ddollar** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;WLxvq0bm&#x2F;361-fix-service-tags-in-refactor"> fix service tags in refactor</a>

**Comment by ddollar** [comment]
Is it possible to add some simple tests to this?

File: manifest/build.go [hunk 1]
```diff
@@ -1,18 +1,37 @@
...
 func (m *Manifest) Build(dir string, s Stream, noCache bool) error {
 	pulls := map[string]string{}
 	builds := []Service{}
 
+	abs, err := filepath.Abs(dir)
+	if err != nil {
+		return err
+	}
+
+	appName := stdcli.ReadSetting("app")
```
Comment (ddollar) [inline manifest/build.go]: This package should be independent of the CLI. App name should be maybe passed in to the function.

Comment (awsmsrc) [inline manifest/build.go]: that is definitely coming!
Firstly the build functions must be unified first

**Comment by awsmsrc** [comment]
This has been merged into #885

**Pull request closed without merging**